src/trg-client.c

```diff
@@ -797,17 +780,15 @@ trg_response *dispatch_public_http(TrgClient *tc, trg_request *req) {
     g_free(cookie_header);
 
     if (headers)
-    	curl_slist_free_all(headers);
-
-    //g_message(response->raw);
+        curl_slist_free_all(headers);
```

Comment:
```c
g_clear_pointer (&headers, curl_slist_free_all);
```

Reply:
Hm, I also should have gotten rid of that comment

Reply:
Oh, I did, that's a diff...